extension-dev.code-workspace

```diff
@@ -64,6 +65,7 @@
       "VSCODE_VERSION": "insiders",
       "ELECTRON_RUN_AS_NODE": null
     },
+    "powershell.cwd": "c:\\Users\\justi\\Projects\\vscode-powershell",
```

Comment:
```suggestion
```

I'd love to figure out a way to not have this happen. It's necessary to support multi-root workspace to know _which_ root (CWD) the user wants to use, but yeah, oops! There's gotta be a way to improve this.